app/views/cases/data_requests/show.html.slim

```diff
@@ -28,7 +28,7 @@ span.visually-hidden
         = t('helpers.fieldset.data_request.request_type')
       p.data_request_request_type
         = t("helpers.label.data_request.request_type.#{@data_request.request_type}")
-        - if @data_request.request_type_note?
+        - if @data_request.display_request_type_note? && @data_request.request_type_note.present?
```

Comment:
Just realised that `display_request_type_note?` is only used in these two places, so it would make sense to incorporate whether there is a note to display into the method.